### PR TITLE
Bug fixs

### PR DESCRIPTION
**Pull request by gaoyan1998**
1. Fix the problem of converting JSON to database storage in the process of obtaining cluster configuration, storage and registration. 
2. Add configuration items that are not needed for cache comparison 
3. Delete some jar parameter residues 
4. Optimize Process to automatically build links 
5. Fix the error that the envid is -1 under the APP 
6. Fix the error of online and offline, and unify the naming

### CODE REVIEW

File: dinky-admin/src/main/java/org/dinky/controller/TaskController.java [hunk 1]
```diff
@@ -168,8 +171,8 @@ public Result<ObjectNode> getJobPlan(@ProcessId @RequestBody TaskDTO taskDTO) {
             dataType = "Task",
             paramType = "body",
             dataTypeClass = Task.class)
```
Comment (Zzm0809) [inline dinky-admin/src/main/java/org/dinky/controller/TaskController.java]: change dataTypeClass to TaskSaveDTO.class

Comment (gaoyan1998) [reply dinky-admin/src/main/java/org/dinky/controller/TaskController.java]: ok

File: dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java [hunk 2]
```diff
@@ -103,7 +103,7 @@ public static void submit(AppParamConfig config) throws SQLException {
     public static String buildSql(AppTask appTask) throws SQLException {
         StringBuilder sb = new StringBuilder();
         // build env task
-        if (Asserts.isNotNull(appTask.getEnvId())) {
+        if (Asserts.isNotNull(appTask.getEnvId()) && appTask.getEnvId() >= 0) {
```
Comment (Zzm0809) [inline dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java]: never == 0
impossible -1 or gt 0

File: dinky-common/src/main/java/org/dinky/data/enums/JobLifeCycle.java [hunk 3]
```diff
@@ -29,7 +29,7 @@
 public enum JobLifeCycle {
     UNKNOWN(0, "未知"),
     DEVELOP(1, "开发"),
-    ONLINE(2, "上线");
+    PUBLISH(2, "上线");
```
Comment (Zzm0809) [inline dinky-common/src/main/java/org/dinky/data/enums/JobLifeCycle.java]: title change to `发布` ?

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Pull request merged**